reference/features/search_parameters.md

```diff
@@ -368,17 +368,26 @@ You would get the following response with **information about the matches in the
 {
   "id": "50393",
   "title": "Kung Fu Panda Holiday",
-  "poster": "https://image.tmdb.org/t/p/w1280/gp18R42TbSUlw9VnXFqyecm52lq.jpg",
+  "poster": "https://image.tmdb.org/t/p/w500/rV77WxY35LuYLOuQvBeD1nyWMuI.jpg",
   "overview": "The Winter Feast is Po's favorite holiday. Every year he and his father hang decorations, cook together, and serve noodle soup to the villagers. But this year Shifu informs Po that as Dragon Warrior, it is his duty to host the formal Winter Feast at the Jade Palace. Po is caught between his obligations as the Dragon Warrior and his family traditions: between Shifu and Mr. Ping.",
   "release_date": 1290729600,
+  "genres": ["Animation", "Family", "TV Movie"],
```

Comment:
Not sure if I should include this. It got added based on example file movies.json.

Reply:
We would probably prefer to leave this off. Even though most of our documentation examples use movies as the index, they aren't intended to be accurate to movies.json. And we do anything we can to make code samples shorter... 🙈

Reply:
Okey, I will remove this change.